src/cljc/rems/common_util.cljc

```diff
@@ -140,3 +154,37 @@
   (is (= [[:a] [:b]] (recursive-keys {:a [1] :b "foo"})))
   (is (= [[:a :b] [:a :c] [:a :d :e] [:a :d :f]]
          (recursive-keys {:a {:b 1 :c nil :d {:e "foo" :f [3]}}}))))
+
+(defn parse-int [s]
+  #?(:clj (try
+            (when s
+              (java.lang.Integer/parseInt s))
+            (catch NumberFormatException e
+              nil))
+     :cljs (let [x (js/parseInt s)]
+             (when-not (js/isNaN x)
+               x))))
+
+
+(deftest test-parse-int
```

Comment:
this ns should probably be added to rems.cljs-tests so that `parse-int` is also tested in js

Reply:
Excellent, didn't know of this.